packages/atomic/src/index.html

```diff
@@ -7,26 +7,28 @@
 
     <script type="module" src="/build/atomic.esm.js"></script>
     <script nomodule src="/build/atomic.js"></script>
+    <link rel="stylesheet" href="/build/atomic.css" />
     <script>
       (async () => {
         await customElements.whenDefined('atomic-search-interface');
         const searchInterface = document.querySelector('#search');
 
         // Initialization
         /* searchInterface.initialize({
-          accessToken: 'my_token',
-          organizationId: 'my_org',
+          accessToken: 'xx564559b1-0045-48e1-953c-3addd1ee4457',
```

Comment:
same token/org as the sample. don't worry 😃

Reply:
Would it be possible to call Headless's `getSampleConfiguration` here?